pkg/kyverno/test/test_command.go

```diff
@@ -324,8 +324,18 @@ func testCommandExecute(dirPath []string, valuesFile string, fileName string, gi
 		}
 
 	} else {
+		var count int
+		var dirTestYamlNameCount int
 		path := filepath.Clean(dirPath[0])
-		errors = getLocalDirTestFiles(fs, path, fileName, valuesFile, rc)
+		errors = getLocalDirTestFiles(fs, path, fileName, valuesFile, rc, &count, &dirTestYamlNameCount)
+
+
+		if count == 0 {
+			fmt.Printf("\n No test yamls found. Please provide test yaml file as kyverno-test.yaml \n")
+		}
+		if dirTestYamlNameCount > 0 {
+			fmt.Printf("\n Note: test.yaml file name is deprecated in 1.6.0 release. Please provide test yaml file as kyverno-test.yaml \n")
```

Comment:
`Note: the test.yaml file name is deprecated in 1.6.0 release. Please use kyverno-test.yaml instead.`